v2/pkg/input/input.go

```diff
@@ -66,9 +66,15 @@ const (
 // Various formats are supported for inputs and their transformation
 func (h *Helper) convertInputToType(input string, inputType inputType, defaultPort string) string {
 	notURL := !strings.Contains(input, "://")
```

Comment:
I think the `notURL` variable should be changed to the opposite (`isURL`) to avoid checking the inverted value over and over.

Reply:
@vzamanillo  , convertInputToType deals with multiple protocols including file protocol , notURL is used while validating file protocol depending on inputType

Reply:
I was talking about this

![imagen](https://user-images.githubusercontent.com/10209695/206543161-a686ecbd-dba9-474d-ba8f-2ee50ab1c456.png)